docs/getting_started.rst

```diff
@@ -31,105 +39,467 @@ example to construct such a store for the local filesystem.
+``table`` and generated a UUID for the partition name.
 
 .. ipython:: python
 
-   dm.tables
-   dm.partitions
+    dm.tables
+    dm.partitions
 
-For each table, ``kartothek`` also tracks the schema of the columns.
-When not specified explicitly on write, it is inferred from the passed data.
-On writing additional data to a dataset, we will also check that the schema
-of the new data matches the schema of the existing data. If it doesn't, we will
-raise an exception.
 
-The schema is a ``pyarrow.Schema`` object which persists the native Arrow types
-for each column. Additionally, the schema also stores infomartion about the Pandas
-types and indices. This information is solely of informative nature and is not
-used by ``kartothek`` itself.
+.. admonition:: A more complex example: multiple tables and partitions
+
+    Sometimes it may be useful to write multiple dataframes with different schemas into
+    a single dataset. This can be achieved by creating a dataset with multiple tables.
+
+    In this example, we create a dataset with two partitions (represented by
+    the dictionary objects inside the list).
+    For each partition, there exist two tables: ``core-table`` and ``aux-table``.
+    The schemas of the tables are identical across partitions.
+
+    .. ipython:: python
+
+       dfs = [
+            {
+                "data": {
+                    "core-table": pd.DataFrame({"col1": ["x"]}),
+                    "aux-table": pd.DataFrame({"f": [1.1]}),
+                },
+            },
+            {
+                "data": {
+                    "core-table": pd.DataFrame({"col1": ["y"]}),
+                    "aux-table": pd.DataFrame({"f": [3.2]}),
+                },
+            },
+       ]
+
+       store_dataframes_as_dataset(store_factory, dataset_uuid="two-tables", dfs=dfs)
+
+
+Reading dataset from storage
+=============================
+
+After we have written the data, we may want to read it back in again. For this we can
+use :func:`kartothek.io.eager.read_table`. This method returns the complete
+table of the dataset as a pandas DataFrame (since there is only a single table in this
+example, it returns the entire dataset).
+
+.. ipython:: python
+
+    from kartothek.io.eager import read_table
+
+    read_table("a_unique_dataset_identifier", store_factory, table="table")
+    
+
+We can also read a dataframe iteratively, using 
+:func:`kartothek.io.iter.read_dataset_as_dataframes__iterator`. This will return a generator
+of dictionaries (one dictionary for each `partition`), where the keys of each dictionary
+represent the `tables` of the dataset. For example,
+
+.. ipython:: python
+
+    from kartothek.io.iter import read_dataset_as_dataframes__iterator
+
+    for partition_index, df_dict in enumerate(
+            read_dataset_as_dataframes__iterator(dataset_uuid="two-tables", store=store_factory)
+        ):
+            print(f"Partition #{partition_index}")
```

Comment:
Is this print rendered by sphinx somehow?

Reply:
Yes.